### ISSUE TITLE
[Other]: IL errors

### ISSUE THREAD

**Issue by OPAmqericaHPE** [AssetRipper/AssetRipper#435]
### Describe the issue

To dumb it down:

In decompiled scripts, any code that interacts with the game itself/changes it in any way throws an IL error. I've been noticing this a ton with interactive scripts.

The screenshot below, it shows that the lines that _should_ be here would be interactable in the game.

![image](https://user-images.githubusercontent.com/46698257/173537986-2953984c-7120-4b8d-87c4-6d54e2b58d86.png)

**Comment by ds5678** [comment]
> General : ScriptContentLevel: Level4

This is normal behavior for script content level 4. You will get lots of bad IL.
